go/kbfs/libkbfs/tlf_journal.go

```diff
@@ -2180,6 +2181,7 @@ func (j *tlfJournal) putBlockData(
 		j.unsquashedBytes += uint64(bufLen)
 	}
 
+	j.config.SubscriptionManagerPublisher().JournalStatusChanged()
```

Comment:
Wouldn't you also need to trigger a notification when data is flushed as well?  Or is that a separate path?

Reply:
I added one here: https://github.com/keybase/client/pull/18380/files#diff-f2721a37acfebd98c14216cf40e39098R1101 please let me know if that's not the correct place!